biomage/account/account.py

```diff
@@ -34,99 +34,124 @@ def generate_password():
     )
 
 
-def create_account(full_name, email, userpool):
+def create_account(full_name, email, aws_profile, region, userpool):
```

Comment:
Is there anty chance you could take the region from the `aws_profile`?

Reply:
Actually, I saw you only use it in `boto3.Session`, which I think doesn't really need to receive region, right?

Reply:
yup, I think it'll use the default region set in `~/.aws/config`